### ISSUE TITLE
[Enhancement] release 📦 Releases on Repository of Chocolatey packages

### ISSUE THREAD

**Issue by 4-FLOSS-Free-Libre-Open-Source-Software** [walterpg/google-drive-sync#54]
**Describe the solution you'd like**
Hello, this is not an issue in itself. Just a suggestion. Would be nice to have an auto update feature for this primary security applications plugin from a trusted source. The SHA256 hash of PLRGX can be verified by installation automatically, optionally if specified.  _For the lazy ones and_ to fix incompatibility issue with future of KeePass version updates with plugin by getting all in one place for example.

I don't know if you have the spare time to maintain in parallel the "packages". Which basically would contain a script to URL of GitHub [latest](../releases/latest) release asset. 

_If you may want to take a look here if you don't know already:
[KeePass' plugins packages](https://community.chocolatey.org/packages?q=keepass-plugin&moderatorQueue=&moderationStatus=all-statuses&prerelease=true&sortOrder=package-download-count)
(There are many plugins and packages count is growing. )_

**Describe alternatives you've considered**
~Another known existing autoupdate integration inside KeePass itself is:
[EarlyUpdateCheck KeePass Plugin](https://github.com/Rookiestyle/EarlyUpdateCheck#one-click-plugin-update) from which this Plugin is not supported yet.~
